modules/doc/content/newsletter/2023/2023_10.md

```diff
@@ -17,8 +12,80 @@ The following syntax can now be used to include other input files in an input fi
 
 See [Includes](input_syntax.md) for more information.
 
+### Arbitrary Lagrangian Eulerian coupling FE and FV
+
+In [PR 25574](https://github.com/idaholab/moose/pull/25574) we demonstrated an
+Arbitrary Lagrangian Eulerian (ALE) laser melt pool simulation in which fluid velocity,
+pressure, and temperature were discretized using finite volumes while
+the mesh displacement field was discretized using finite elements. This required
+a few important developments in MOOSE:
+
+- Addition of `NodeArg` overloads to the MOOSE functor (on-the-fly evaluation)
+  system. This allows coupling the finite volume velocity variables into the Dirichlet
+  boundary condition that displaces the surface in the ALE simulation
+- Implementation of "finite-volume" functor methods for finite element variables
+  for coupling finite element variables into the finite volume discretization
+  (`ElemArg`, `FaceArg`)
+- Allowing finite volume discretizations to operate on a displaced mesh
+
+### Hybrid CG-DG formulation for Navier-Stokes
+
+We merged [PR 23986](https://github.com/idaholab/moose/pull/23986) which
+demonstrates a hybrid-method for discretizing the Navier-Stokes equations in
+which the pressure uses a continuous Galerkin discretization while the velocity
+components use a discontinuous Galerkin discretization. This formulation is
+intrinsically
+[LBB stable](https://en.wikipedia.org/wiki/Ladyzhenskaya%E2%80%93Babu%C5%A1ka%E2%80%93Brezzi_condition)
+and allows a natural upwind stabilization for momentum advection. For more
+information regarding the method, please see [hcgdgfe.md].
+
+### Module rename
+
+The Heat Conduction module was renamed to Heat Transfer to better match the capabilities of the module.
+
 ## libMesh-level Changes
```

Comment:
```suggestion
```
This section should be removed. As I don't think we've had a libmesh update this month - @roystgnr correct if wrong.